app/routes/admin/courts/remove.js

```diff
@@ -0,0 +1,52 @@
+const { routes, views } = require('../../../constants/admin')
+const { admin } = require('../../../auth/permissions')
+const ViewModel = require('../../../models/common/single-submit-autocomplete')
+
+module.exports = [
+  {
+    method: 'GET',
+    path: `${routes.removeCourt.get}`,
+    options: {
+      auth: { scope: [admin] },
+      handler: async (request, h) => {
+        const backLink = routes.courts.get
+
+        return h.view(views.addAdminRecord, new ViewModel({
+          backLink,
+          recordTypeText: 'court',
```

Comment:
It might make sense to have this construct in /constants as it's referred to a few times. And then we (almost) just need to change the constant name for different routes/methods